app/src/main/java/com/hover/stax/channels/ChannelsActivity.java

```diff
@@ -63,12 +63,12 @@ private void goToChannelSelection() {
 		findViewById(R.id.continue_btn).setVisibility(View.VISIBLE);
 	}
 
-	private void onSelectedUpdate(List<Integer> ids) {
-		if (ids.size() > 0) {
+	private void validateToLogeventThenSaveAndContinue(List<Integer> channelIds) {
```

Comment:
This is an improvement, but the really right thing would be to name this function something like `changeContinueClickAction`, then move the actual click action to their own functions as well, something like `findViewById(R.id.continue_btn).setOnClickListener(view -> logChoicesSaveAndContinue)`

Reply:
Yup, you're right